data/mods/Magiclysm/Spells/animist.json

```
@@ -518,5 +518,56 @@
     "energy_source": "MANA",
     "difficulty": 3,
     "damage_type": "pure"
+  },
+  {
+    "id": "devour",
+    "type": "SPELL",
+    "name": "Devour",
+    "description": "Kill a summoned creature, consuming it life energy and regain some health.  Each level increase the amount of restored HP",
```

```suggestion
    "description": "Kill a summoned creature, consuming its life energy to regain some health.",
```
Players already see scaling when they study, I considered it redundant.